libopenage/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ find_path(FREETYPE_INCLUDE_DIRS freetype/freetype.h HINTS /usr/include/freetype2
 
 # provide apple qt5 location
 if(APPLE)
-	list(APPEND CMAKE_PREFIX_PATH /usr/local/opt/qt5)
+	list(APPEND CMAKE_PREFIX_PATH ${HOMEBREW_PREFIX}/opt/qt5)
 endif()
```

Comment:
how is this variable available here already? probably the `brew --prefix` invocation has to be called in this `if(APPLE)` block and not down below?